internal/cli/cmd/hubble.go

```diff
@@ -62,8 +61,8 @@ func newCmdHubbleEnable() *cobra.Command {
 
 	cmd.Flags().StringVarP(&params.Namespace, "namespace", "n", "kube-system", "Namespace Cilium is running in")
 	cmd.Flags().BoolVar(&params.Relay, "relay", true, "Deploy Hubble Relay")
-	cmd.Flags().StringVar(&params.RelayImage, "relay-image", defaults.RelayImage, "Image path to use for Relay")
```

Comment:
Reviewer note: The `defaults.RelayImage` is still passed to `BuildImagePath` above and therefore will be used if this flag is absent.